ts/screens/profile/components/services/ServicesContactComponent.tsx

```diff
@@ -1,114 +1,73 @@
-    mode: ServicesPreferencesModeEnum.AUTO,
-    description1: I18n.t("services.optIn.preferences.quickConfig.body.text1"),
-    description2: I18n.t("services.optIn.preferences.quickConfig.body.text2")
+    value: I18n.t("services.optIn.preferences.quickConfig.title"),
+    id: ServicesPreferencesModeEnum.AUTO,
+    description: (
+      <LabelSmall color="grey-700" weight="Regular">
+        {I18n.t("services.optIn.preferences.quickConfig.body.text1")}{" "}
+        <Text style={{ color: IOColors["grey-700"], fontWeight: "600" }}>
+          {I18n.t("services.optIn.preferences.quickConfig.body.text2")}
+        </Text>
+      </LabelSmall>
+    )
   },
   {
-    title: I18n.t("services.optIn.preferences.manualConfig.title"),
-    mode: ServicesPreferencesModeEnum.MANUAL,
-    description1: I18n.t("services.optIn.preferences.manualConfig.body.text1")
+    value: I18n.t("services.optIn.preferences.manualConfig.title"),
+    id: ServicesPreferencesModeEnum.MANUAL,
+    description: I18n.t("services.optIn.preferences.manualConfig.body.text1")
   }
 ];
 
-const ServicesContactComponent = (props: Props): React.ReactElement => {
-  const renderListItem = ({ item }: ListRenderItemInfo<ContactOption>) => {
-    const isSelected = item.mode === props.mode;
-    return (
-      <>
-        <TouchableDefaultOpacity
-          style={[
-            IOStyles.row,
-            {
-              justifyContent: "space-between"
-            }
-          ]}
-          accessibilityRole={"radio"}
-          accessibilityState={{ checked: isSelected }}
-          onPress={
-            // do nothing if it is the current mode set
-            isSelected ? constNull : () => props.onSelectMode(item.mode)
-          }
-        >
-          <View style={IOStyles.flex}>
-            {props.showBadge &&
-              item.mode === ServicesPreferencesModeEnum.AUTO && (
-                <IOBadge
-                  small
-                  variant="solid"
-                  color="blue"
-                  text={I18n.t("services.optIn.preferences.oldUsersBadge")}
-                />
-              )}
-            <H4>{item.title}</H4>
-            <H5 weight={"Regular"}>
-              {item.description1}
-              {item.description2 && <H5>{`\n${item.description2}`}</H5>}
-            </H5>
-          </View>
-          <HSpacer size={24} />
-          <View style={{ alignSelf: "flex-start" }}>
-            <Icon
-              name={isSelected ? "legRadioOn" : "legRadioOff"}
-              color="blue"
-              size={24}
-            />
-          </View>
-        </TouchableDefaultOpacity>
-        <VSpacer size={16} />
-        <ItemSeparatorComponent noPadded />
-        <VSpacer size={16} />
-      </>
-    );
+const ServicesContactComponent = (props: Props): ReactElement => {
+  const { mode, onSelectMode } = props;
+  const [selectedItem, setSelectedItem] = useState(mode);
+  const prevMode = usePrevious(mode);
+
+  const handlePress = (value: any) => {
```

Comment:
What about using a callback? What do you think?

Reply:
done!